database/queries.ts

```diff
@@ -134,6 +134,43 @@ function calculateStreak(entryTimes: number[], currentTime :number, waitTime :nu
   return streak;
 }
 
+export const readInventory = async () => {
+  const db = await dbPromise;
+  const rows : {InventoryOfItems:string}[]= await db.getAllAsync('SELECT InventoryOfItems FROM App');
+  console.log(rows);
+
+  const inventoryString = rows[0].InventoryOfItems;
+  const inventory = JSON.parse(inventoryString);
+
+  return(inventory);
+}
+
+export const updateInventory = async (inventory: ItemAndCount[]) => {
+  const db = await dbPromise;
+  const result = await db.runAsync('UPDATE App SET InventoryOfItems = ?', JSON.stringify(inventory));
+  console.log(`Updated ${result.changes} row(s)`);
+}
+
+export const getCurrencyAmount = async () => {
+  const db = await dbPromise;
+  const rows: {CurrencyAmount: number}[] = await db.getAllAsync('SELECT CurrencyAmount FROM App');
+  return rows[0].CurrencyAmount;
+}
```

Comment:
I think lets get rid of the currency stuff for now. If we really want to do this, the best way would be record transactions so that we can derive current currency, etc.

However, it's more work and I think it's not needed.